apps/analysis_framework/migrations/0041_widget_mapping.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 3.2.17 on 2024-02-22 05:02
+
+from django.db import migrations,transaction
+
+@transaction.atomic
+def analysis_framework_widgets_mapping(apps, schema_editor):
+    AnalysisFramework = apps.get_model('analysis_framework', 'AnalysisFramework')
+    analysis_framework_data = AnalysisFramework.objects.filter(properties__isnull=False)
+    for af in analysis_framework_data:
+        if af.properties != {}:
+            widget_properties = af.properties
+            widget_properties['stats_config']['widget_1d'] = widget_properties['stats_config'].get('widget1d') or widget_properties['stats_config'].get('widget_1d') or []  # noqa
+            widget_properties['stats_config']['widget_2d'] = widget_properties['stats_config'].get('widget2d') or widget_properties['stats_config'].get('widget_2d') or []  # noqa
+            widget_properties['stats_config']['organigram_widgets'] = widget_properties['stats_config'].get('affected_groups_widgets') or widget_properties['stats_config'].get('organigram_widgets') or widget_properties['stats_config'].get('organigram_widget') or []  # noqa
+            widget_properties['stats_config']['multiselect_widgets'] = widget_properties['stats_config'].get('specific_needs_groups_widgets') or widget_properties['stats_config'].get('multiselect_widgets') or widget_properties['stats_config'].get('multiselect_widget') or []  # noqa
```

Comment:
Let's confirm if we need to convert legacy key/value to array.

Reply:
``` widet_1d, widget_2d,multiselect_widget,organigram_widet are in arry ```